tests/functional/rbac/test_permissions_deleting.py

```diff
@@ -0,0 +1,152 @@
+    Test that user loses access if user removed from group with policy
+    """
+    cluster_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    group = sdk_client_fs.group_create("test_group", user=[{"id": user.id}])
+    create_policy(sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[group])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+    group.update(user=[])
+    is_denied(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_remove_object_from_policy(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user loses access if object changed from policy
+    """
+    cluster_via_admin, service_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    policy = create_policy(sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+    policy.update(object=[{"id": service_via_admin.id, "type": "service"}])
+    is_denied(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_change_child_role(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user loses access if change policy role
+    """
+    cluster_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    policy = create_policy(sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+
+    role = sdk_client_fs.role_create(
+        name="Another role",
+        display_name="Another role",
+        parametrized_by=["cluster", "service", "component", "host", "provider"],
+        child=[{"id": sdk_client_fs.role(name=BusinessRoles.ViewImports.value.role_name).id}],
+    )
+    policy.update(role={"id": role.id})
+    is_allowed(cluster, BusinessRoles.ViewImports)
+    is_denied(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_remove_user_from_policy_but_still_in_group(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user is still have access if user removed from policy but is in group
+    """
+    cluster_via_admin, *_ = prepare_objects
+    cluster = user_sdk.cluster(id=cluster_via_admin.id)
+    group = sdk_client_fs.group_create("test_group", user=[{"id": user.id}])
+    policy = create_policy(
+        sdk_client_fs, BusinessRoles.ViewConfigurations, objects=[cluster], users=[user], groups=[group]
+    )
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+    policy.update(user=[{"id": sdk_client_fs.user(username="admin").id}])
+    is_allowed(cluster, BusinessRoles.ViewConfigurations)
+
+
+def test_remove_user_from_group_but_still_in_another_group(user_sdk: ADCMClient, user, prepare_objects, sdk_client_fs):
+    """
+    Test that user is still have access if user removed from group but is in another group
```

Comment:
From the test body, it looks like the user isn't removed from the group. The group is removed from the policy. Please, clarify

Reply:
fixed